### DISCUSSION TITLE
어서오세요~ Team4-BF2 Discussions입니다!

### DISCUSSION THREAD

**Discussion by jcw1031** [woowa-techcamp-2024/Team4-BF2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hek-git** [comment]
안녕하세요~
